service_test.go

```diff
@@ -143,427 +144,433 @@ var mockTransactions = []Nip47Transaction{
 }
 var mockTransaction = &mockTransactions[0]
 
-// TODO: split up into individual tests
-func TestHandleEvent(t *testing.T) {
+func TestBackwardsCompatibility(t *testing.T) {
+	// Test without adding a single permission
+}
+
+func TestHandleGetBalanceEvent(t *testing.T) {
```

Comment:
Should we split the tests into separate files based on NIP-47 method?

Reply:
Maybe after splitting into `package integration_tests`? Right now it would become too cluttered if we can't make a folder in the root

Reply:
Sounds good, thanks